mne/surface.py

```diff
@@ -333,8 +333,8 @@ def complete_surface_info(surf, do_neighbor_vert=False, copy=True,
     if len(idx) > 0:
         logger.info('    Vertices [%s] have fewer than three neighboring '
                     'tris, omitted' % ','.join([str(ii) for ii in idx]))
-    for k in idx:
-        surf['neighbor_tri'] = np.array([], int)
+        for k in idx:
+            surf['neighbor_tri'][k] = np.array([], int)
```

Comment:
It only manifested as "unused variable" in my IDE :) it should only affect borderline cases in which the geometry is defective to start with (`idx = np.where([len(n) < 3 for n in surf['neighbor_tri']])[0]`)